ngraph/frontend/tensorflow/src/decoder_proto.hpp

```diff
@@ -4,13 +4,13 @@
 
 #pragma once
 
-#include <ngraph/ngraph.hpp>
 #include <string>
-#include <tensorflow_frontend/decoder.hpp>
 #include <vector>
 
 #include "attr_value.pb.h"
+#include "ngraph/ngraph.hpp"
```

Comment:
please check if it is needed. Propose to avoid

Reply:
fixed